mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -141,6 +143,27 @@ func stop() error {
 			return err
 		}
 
+		//getting service endpoint
+		var opts meshkitkube.ServiceOptions
+		opts.Name = "meshery"
+		opts.Namespace = utils.MesheryNamespace
```

Comment:
Perhaps, we should have a central struct that contains this. Thoughts @layer5io/mesheryctl-maintainers?

Reply:
@kumarabd do the CRs first need to be deleted as a graceful shutdown?

Reply:
Yes they will have to be deleted @lee

Reply:
@leecalcote we could move "meshery" to a global variable similar to `utils.MesheryNamespace`. But if a `struct` would be cleaner, then we can refactor things to make that change.